build.rs

```diff
@@ -23,28 +23,62 @@
 
 extern crate gcc;
 
+use std::env;
+use std::path::PathBuf;
+
+const ANDROID_AARCH64_COMPILER: &'static str = "toolchains/aarch64-linux-android-4.9/prebuilt/darwin-x86_64/bin";
+const ANDROID_ARM_COMPILER: &'static str = "toolchains/arm-linux-androideabi-4.9/prebuilt/darwin-x86_64/bin";
```

Comment:
What about other platforms? It will not really work on anything else the `darwin-x86_64` right?